lib/notifier.js

```diff
@@ -423,6 +429,26 @@ function addItem(item, callback) {
 
```

Comment:
Yes, good catch!  I was assuming that buildItemData caught its own exceptions, but you're right.